apps/site/components/EOL/VulnerabilitiesTable/index.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames';
+import { useTranslations } from 'next-intl';
+import type { FC } from 'react';
+
+import VulnerabilityChip from '#site/components/EOL/VulnerabilityChips/Chip';
+import LinkWithArrow from '#site/components/LinkWithArrow';
+import type { Vulnerability } from '#site/types/vulnerabilities';
+
+const VulnerabilitiesTable: FC<{
+  vulnerabilities: Array<Vulnerability>;
+  maxWidth?: string;
+}> = ({ vulnerabilities, maxWidth = 'max-w-2xs' }) => {
+  const t = useTranslations();
+
+  return (
+    <table className="w-full">
+      <thead>
+        <tr>
+          <th>{t('components.eolModal.table.cves')}</th>
+          <th>{t('components.eolModal.table.severity')}</th>
+          <th>{t('components.eolModal.table.overview')}</th>
+          <th>{t('components.eolModal.table.details')}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {vulnerabilities.map((vuln, i) => (
```

Comment:
nit: call this `vulnerability`

Reply:
addressed via https://github.com/nodejs/nodejs.org/pull/7990/commits/9fe21ccb59cfddd112b4aa0020e2aa23774b1dfc